package.json

```diff
@@ -29,7 +29,7 @@
         "lint": "npm run lint:less && npm run lint:css && npm run lint:js",
         "lint:less": "stylelint docs/src/less src/less --config .stylelintrc-less",
         "lint:css": "stylelint dist --config .stylelintrc-css",
-        "lint:js": "eslint 'docs/src/js/base/*.js' > lint.log",
+        "lint:js": "eslint 'docs/src/js/base/*.js' > lint.log && eslint 'src/less/**/stories/*.js' > lint.log ",
```

Comment:
Only quick nit...
By doing it this way, it will override the lint.log file for the second time and won't keep the data from the first. 
You should change it to this:
`eslint 'docs/src/js/base/*.js' > lint.log && eslint 'src/less/**/stories/*.js' >> lint.log` (double `>>` appends to the previous one, while `>` clears the log file and then appends)

Reply:
good catch, thanks! Fixed